lib/request_handler.rb

```diff
@@ -17,11 +18,16 @@ def configuration
       @configuration ||= ::Confstruct::Configuration.new do
         logger Logger.new(STDOUT)
         separator '__'
+        validation_backend Validation::DryBackend
```

Comment:
hmm not a fan of the term `backend` in this context

How about calling it `validation_engine` ?

Reply:
we can do that